mantidproject/mantidimaging
# Recon Window: Cor Table automatically does fit

Fixes #659 

To test:
- Load Data
- Clean the data (ROI Norm, Crop, Remove Outliers and Flat fielding)
- Open Reconstruction window
- Ensure each time the table changes that the fit is performed
